updater/lib/dependabot/updater/operations/refresh_security_update_pull_request.rb

```diff
@@ -197,6 +213,24 @@ def check_and_update_pull_request(dependencies)
         # rubocop:enable Metrics/MethodLength
         # rubocop:enable Metrics/CyclomaticComplexity
 
+        # Feature testing to fix issues related with scenarios when a PR is updated while containing
+        # the same version in existing PR and dependeabot updater closes and raised same PR
+        sig { params(lead_dep_name: String, existing_pr_lead_dep_ver: String).returns(T::Boolean) }
+        def pr_lead_dep_latest_ver(lead_dep_name, existing_pr_lead_dep_ver)
```

Comment:
I think a nicer name for this method would be `pr_exists_for_dependency_version?`

Reply:
this code is no longer relevant